lib/viewRenderer/relationshipRendering/RelationshipBuilder.ts

```diff
@@ -0,0 +1,194 @@
+import { shapes, dia } from 'jointjs';
+import { RelationshipAttributesBuilder } from '@lib/viewRenderer/relationshipRendering/RelationshipAttributesBuilder';
+import { EdgePointerBuilder } from '@lib/viewRenderer/relationshipRendering/EdgePointerBuilder';
+import { ViewSetting } from '@lib/model/ViewSetting';
+import { RelationshipType } from '@lib/common/enums/relationshipType';
+
+interface BaseRelationshipSettings {
+  type: string;
+  isBidirectional: boolean;
+}
+
+interface RelationshipSettings extends BaseRelationshipSettings {
+  relationshipModelId: string;
+  relationshipViewId: string;
+  bendpoints: Array<{ x: number; y: number }>;
```

Comment:
Hi, @rodrigolimoes . Do you think it would be a good idea to extract another type for { x: number; y: number }?

Reply:
Hi, @Diorbert.
Yes, I think it would be a good idea.

Reply:
I fixed it